src/sass/components/_hero-header.scss

```diff
@@ -0,0 +1,67 @@
+.container--header-bg {
+  background-color: $pinkBg;
+}
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 6px;
+  &__hamburger {
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+  }
+}
+.container {
+  overflow: clip;
+  position: relative;
+}
+.hero {
+  background-color: $pinkBg;
+  margin-top: -1px;
+  margin-bottom: -1px;
+  overflow: clip;
```

Comment:
Z tego co czytałem overflow nie ma wartości clip. Co to robi?

Reply:
Aktualnie zmieniłem na hidden, bawiłem się nieco bo clip może mi się przydać przy sekcji z tabletem, ale to jeszcze zobaczę.
Jak masz ochotę to prosze:https://kilianvalkhof.com/2022/css-html/do-you-know-about-overflow-clip/